src/examples.md

```diff
@@ -1071,7 +1071,7 @@ We recommend viewing examples 13 and 31 before viewing this example._
 <img class="floatright" width="280pt" src="../img/examples/ex33.png">
 
 This example code demonstrates the use of MFEM to solve the fractional Laplacian problem
-$$ (-\Delta)^\alpha u = 1, \quad 0 < \alpha < 1, $$
+$$ (-\Delta)^\alpha u = 1, \quad 0 < \alpha, $$
```

Comment:
I think that the general consensus is that $\alpha > 0$ looks better than $0 < \alpha$, so would write it that way instead.